.travis.yml

```diff
@@ -6,6 +6,8 @@ matrix:
     include:
       - env: TEST_SUITE=danger 
       - env: TEST_SUITE=jsfm
+      - env: TEST_SUITE=ios
```

Comment:
No need to add `os: osx`?

Reply:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/33521539/61276574-e6e92f80-a7e2-11e9-8941-6bb641696e37.png">

Reference link:https://www.objc.io/issues/6-build-tools/travis-ci/